src/passes/stringify-walker.h

```diff
@@ -115,15 +115,18 @@ struct HashStringifyWalker : public StringifyWalker<HashStringifyWalker> {
   // After calling walkModule, this vector contains the result of encoding a
   // wasm module as a string of uint64_t values. Each value represents either an
   // Expression or a separator to mark the end of control flow.
-  std::vector<uint64_t> hashString;
+  std::vector<unsigned> hashString;
```

Comment:
Can we use `uint32_t` instead of `unsigned`? The extra explicitness is nice, and they should be compatible on all systems we build on.

Reply:
done